submissions/maxim-zabolotny/HTML-CSS-Popup/style.css

```diff
@@ -0,0 +1,180 @@
+    -webkit-appearance: none;
+    appearance: none;
+  }
+  
+  header {
+    text-align: right;
+    margin-right: 1rem;
+    margin-top: 1rem;
+  }
+  
+  .user_menu {
+    display: inline-flex;
+    align-items: center;
+    justify-content: space-around;
+    font-size: 0;
+    list-style: none;
+  }
+  
+  .apps_menu {
+    display: none;
+    position: absolute;
+    top: 5.2rem;
+    right: 6rem;
+    width: 320px;
+    height: 480px;
+    border-radius: 5px;
+    box-shadow: 5px 5px 10px rgb(3, 3, 3);
+    text-align: center;
+    font-size: 12px;
+  }
+  
+  .apps_menu::after {
+    content: '';
+    position: absolute;
+    top: -11px;
+    right: 60px;
+    width: 20px;
+    height: 20px;
+    border: none;
+    background-color: white;
+    border-top: solid 1px rgb(96, 152, 243);
+    border-right: solid 1px rgb(96, 152, 243);
+    transform: rotate(-45deg);
+    z-index: 2;
+    clip-path: polygon(-1px 0px, 21px 0px, 21px 22px);
+  }
+  
+  .user_menu__element {
+    margin: 10px;
+    cursor: pointer;
+  }
+  
+  .user_menu__element:hover {
+    outline: 2px solid #6098f3;
+    outline-offset: 4px;
+  }
+  
+  .user_menu__element-text {
+    font-size: 14px;
+  }
+  
+  .user_menu__element-anchor {
+    display: block;
+  }
+  
+  .apps_menu__checkbox:focus + .user_menu__element-anchor {
+    outline: 2px solid #6098f3;
+    outline-offset: 4px;
+  }
+  
+  .user_menu__user_photo {
+    border-radius: 50%;
+    width: 3rem;
+    height: 3rem;
+  }
+  
+  .apps_menu__checkbox:checked ~ .apps_menu {
+    display: block;
+  }
+
+  .bell_img {
+      width: 2.5rem;
+      height: 2.5rem;
+  }
+  
+  .apps_menu__content {
+    height: 480px;
+    overflow-y: auto;
+  }
+  
+  .first_page {
+    height: 448px;
+  }
+  
+  .more_apps__checkbox:focus + .more_apps__label {
+    background-color: rgb(31, 111, 241);
+  }
+  
+  .more_apps__label {
+    display: block;
+    height: 32px;
+    line-height: 32px;
+    background-color: rgb(272, 228, 167);
+    cursor: pointer;
+  }
+  
+  .more_apps__label:hover {
+    background-color: rgb(31, 111, 241);
+  }
+  
+  .more_apps__checkbox:checked + .more_apps__label {
+    display: none;
+  }
+  
+  .more_apps__checkbox:checked ~ .second_page {
+    display: block;
+  }
+  
+  .second_page {
+    display: none;
+    border-top: solid 1px #ECEEF0;
+  }
+  
+  .second_page__footer {
+    padding: 20px;
+  }
+  
+  .icon_card {
+    display: inline-flex;
+    flex-direction: column;
+    width: 80px;
+    height: 90px;
+    margin: 7px;
+    align-items: center;
+    justify-content: space-around;
+    border: solid 1px transparent;
+    cursor: pointer;
+  }
+  
+  .icon_card:hover {
+    background: #E8F0FE;
+    border-radius: .5rem;
+  }
+  
+  .icon_card:focus {
+    margin: 6px;
+    outline: none;
+    border: solid 2px rgb(96, 152, 243);
+    border-radius: 3px;
+  }
+  
+  .icon_card img {
+    max-width: 48px;
+    min-height: 48px;
+    object-fit: contain;
+  }
+  
+ 
```

Comment:
if you see this symbol on github at the end of the file, it means that empty line is needed at the end.
![empty line](https://user-images.githubusercontent.com/10871447/101401867-9db7e500-38db-11eb-85b7-dfa605951e2f.png)
https://github.com/kottans/frontend-2021-homeworks/pull/232/files
Could you please add an empty line but without any spaces (you added two new lines but with space, Github reads this as a symbol))

Reply:
All fixed
Thanks for your help